.canon.yaml

```diff
@@ -0,0 +1,7 @@
+viamrtsp:
+    image_amd64: ghcr.io/viamrobotics/canon:amd64
```

Comment:
Should we be using antique in order to ensure the resulting binaries are libc compatible on older systems?

Reply:
Bullseye:
```
ldd --version
ldd (Debian GLIBC 2.31-13+deb11u5) 2.31
```
Ubuntu 18.04 (antique):
```
ldd --version
ldd (Ubuntu GLIBC 2.27-3ubuntu1.6) 2.27
```

Makes sense to use antique to support `GLIBC>=2.27`.

Also, had to add in an explicit `PKG_CONFIG_PATH` setter for pc files to be picked up in antique container.